versions.json

```diff
@@ -1,5 +1,61 @@
 {
   "versions": [
+    {
```

Comment:
Sorry for late review, if I didn't missed something, it seems we should move `python3-java17`, `python3`, `latest`, `r`, `scala` move from 3.5.5 to 4.0.0.

Reply:
Here is the follow-up.
- https://github.com/apache/spark-docker/pull/86